app/helpers/blacklight/blacklight_helper_behavior.rb

```diff
@@ -110,7 +110,9 @@ def document_has_value? document, field_config
   # @param [Blacklight::Solr::Response] response
   # @return [Boolean]
   def should_show_spellcheck_suggestions? response
-    response.total <= spell_check_max && response.spelling.words.any?
+    response.total <= spell_check_max &&
+      !response.spelling.nil? &&
```

Comment:
Please add a comment that states when this happens. (e.g. not using the Solr repository adapter)

Reply:
will do.